react-components/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,7 @@
+export default function CustomButton() {
```

Comment:
I suggest you always add a TypeScript return type on your functions. For components, use JSX.Element:
`export default function CustomButton(): JSX.Element {...}`